arlo-client/src/components/AuditForms/ResetButton.test.tsx

```diff
@@ -39,7 +52,53 @@ describe('ResetButton', () => {
 
     await wait(() => {
       expect(apiMock).toBeCalledTimes(1)
+      expect(toasterMock).toBeCalledTimes(1)
       expect(updateAuditMock).toBeCalledTimes(1)
     })
   })
+
+  it('handles server errors', async () => {
+    toasterMock.mockImplementationOnce(() => true)
+    const updateAuditMock = jest.fn()
+    const wrapper = document.createElement('div')
+    wrapper.setAttribute('id', 'reset-button-wrapper')
+
+    const { getByText } = render(
+      <ResetButton electionId="1" updateAudit={updateAuditMock} />,
+      { container: document.body.appendChild(wrapper) }
+    )
+
+    fireEvent.click(getByText('Clear & Restart'), { bubbles: true })
+
+    await wait(() => {
+      expect(apiMock).toBeCalledTimes(1)
+      expect(toasterMock).toBeCalledTimes(1)
+      expect(updateAuditMock).toBeCalledTimes(0)
+      expect(toastSpy).toBeCalledTimes(0)
+    })
+  })
+
+  it('handles 404 errors', async () => {
+    toasterMock.mockImplementationOnce(() => true)
```

Comment:
I'm not sure whether you're trying to standardize on just using `mockImplementation(Once)`, but if not you can use the other helpers like `mockReturnValueOnce(true)` or `mockRejectedValueOnce(new Error('404'))`.